### PR TITLE
Fix some lint errors

### PR DESCRIPTION
**Pull request by fejta**

### CODE REVIEW

File: util/gcs/client.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ type Stater interface {
 	Stat(ctx context.Context, prefix Path) (*storage.ObjectAttrs, error)
 }
 
-// A copier can cloud copy an object to a new location if it
+// A Copier can cloud copy an object to a new location if it
```
Comment (chases2) [inline util/gcs/client.go:58]: if it... what?

**Review by chases2 — APPROVED** [review]
Looks good to me; one incomplete comment that might be worth fixing
/hold

File: util/gcs/client.go (hunk 1 above)
Comment (fejta) [reply util/gcs/client.go]: Done

**Pull request merged**